src/sections/CategorySection.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from '@emotion/styled'
+import { categoryList } from '@/mockdata/categoryList'
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 0;
+`
+const Title = styled.h1`
+  ${({ theme }) => `
+    color: ${theme.colors.textDefault};
+    font-size: ${theme.typography.title1Bold.fontSize};
+    font-weight: ${theme.typography.title1Bold.fontWeight};
+    line-height: ${theme.typography.title1Bold.lineHeight};
+  `}
+  align-self: flex-start;
+  margin-left: 20px;
+`
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  gap: 10px 40px;
+`
+const CategoryItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+`
+const CategoryImage = styled.img`
+  width: 60%;
+`
+const CategoryName = styled.span`
+
+`
+
+const CategorySection = () => {
+  return (
+    <Container>
+      <Title>선물 테마</Title>
+      <Grid>
+        {categoryList.map((category)=>(
+          <CategoryItem key = {category.themeId}>
+            <CategoryImage src = {category.image}/>
```

Comment:
요런 부분은 보통 붙여서 사용하는데, 혹시 prettier 설정이 잘 되어있는지, 그리고 format on save가 잘 동작하고 있는지 확인해보실 수 있을까용? 허헛ㅎㅎ

```suggestion
          <CategoryItem key={category.themeId}>
            <CategoryImage src={category.image}/>
```

Reply:
prettier가 자동으로 적용되고 있지 않은 것 같아서 script에 수동으로 명령어 추가해서 prettier 적용 확인했는데 혹시 자동으로 적용할 수 있는 방법이 있을까요>??

Reply:
이 부분 해결했습니다! 
default formatter가 prettier로 설정되어있지 않아서 자동으로 수정되지 않았습니다.
이제 자동 적용 되어요 :)

Reply:
좋습니당! 꺄아 👍 👍